includes/handler/class-create.php

```diff
@@ -21,6 +21,13 @@ public static function init() {
 			10,
 			3
 		);
+
+		\add_filter(
```

Comment:
@pfefferle Am I getting something wrong or is this filter added every time? That would also mean, that if one adds a custom handler for a `Create` activity one must also add a custom validation function that completely overrides (is applied later, higher priority) the one in this class (if for example one does not need `inReplyTo` set).

Reply:
good point!!!

Reply:
Maybe I am reading this wrong but I am listening on `activitypub_inbox_create` which should still be called normally, no?

https://github.com/akirk/friends/blob/199af50ae54ee82e8cc77b2311a7149fb1eea8b2/feed-parsers/class-feed-parser-activitypub.php#L45

Reply:
@akirk This is what I did as well (I was actually studying your code, before starting implementing my event-import feature). But the "aggressive" filter here makes incoming `Create` activities not even reach the handler in `activitypub_inbox_create` when the `object` within the `Create` does not have `inReplyTo` set.